pipit/tests/config.py

```diff
@@ -0,0 +1,47 @@
+import pipit as pp
+
```

Comment:
Is there a function in which we might want to use global config? For example, in Hatchet we have the `tree` function that prints the CCT. One of the parameters it takes is `colormap`. We want to change colormap globally using `set_option`. Do we have a function like this in Pipit? If yes, can we add a test that it works?

Reply:
There is a `set_option` function that is exposed to users. I'm not sure if this is what you mean.

Reply:
In Hatchet, if we specify a `colormap` parameter for the `tree` function, does it set the `colormap` globally? Or just once, for that specific tree?

Reply:
No, it just changes the colormap for that call. If we want to change it globally, we have to use set_option.

Reply:
That makes sense, we should be able to do the same in Pipit for timeline, etc.

Reply:
Yes, I was asking if we could add a test for something like that.

Reply:
If I understand what you're getting at, the timeline function would say something like:

```
def timeline( ..., colormap = None):
    ...
    if colormap = None:
        colormap = pp.get_config("colormap")
    ...
```

We currently have tests for `pp.get_config`. I think a specific test for the code above would be in the timeline test? Please correct me if I'm misunderstanding anything.

Reply:
Hmm, I think that makes sense. Let's discuss this with @bhatele later.